src/applications/disability-benefits/all-claims/pages/unemployabilityFormUpload.js

```diff
@@ -0,0 +1,62 @@
+import fileUploadUI from 'us-forms-system/lib/js/definitions/file';
+import environment from '../../../../platform/utilities/environment';
+
+import { DocumentDescription } from '../content/uploadFormDocuments';
+
+const FIFTY_MB = 52428800;
+
+export const uiSchema = {
+  'ui:title': 'Upload VA Form 21-8940',
+  'ui:description': DocumentDescription,
+  iu8940: fileUploadUI('', {
+    itemDescription: 'Document Type (VA Form 21-8940)',
+    hideLabelText: true,
+    fileUploadUrl: `${environment.API_URL}/v0/upload_supporting_evidence`,
+    fileTypes: ['pdf', 'jpg', 'jpeg', 'png'],
+    maxSize: FIFTY_MB,
+    createPayload: file => {
+      const payload = new FormData();
+      payload.append('supporting_evidence_attachment[file_data]', file);
+
+      return payload;
+    },
+    parseResponse: (response, file) => ({
+      name: file.name,
+      confirmationCode: response.data.attributes.guid,
+    }),
+    // this is the uiSchema passed to FileField for the attachmentId schema
+    // FileField requires this name be used
+    attachmentSchema: {
+      'ui:title': 'Document type',
+    },
+    // this is the uiSchema passed to FileField for the name schema
+    // FileField requires this name be used
+    attachmentName: {
+      'ui:title': 'Document name',
+    },
+  }),
+};
+
+export const schema = {
+  type: 'object',
+  required: ['iu8940'],
+  properties: {
+    iu8940: {
```

Comment:
:thinking: What's the `iu` stand for?

Reply:
Individual Unemployability. Sort of a mouthful for a form property.